src/ngResource/resource.js

```diff
@@ -667,7 +671,18 @@ angular.module('ngResource', ['ng']).
               extend({}, extractParams(data, action.params || {}), params),
               action.url);
 
-            var promise = $http(httpConfig).then(function(response) {
+            // Start the promise chain
+            var promise;
+            if (requestInterceptor) {
```

Comment:
Maybe it would be better to be consistent and do the following regardless if there re request interceptors or not:

```js
var promise = $q.
  resolve(httpConfig).
  then(requestInterceptor).
  catch(requestErrorInterecptor).
  then($http);
```

<sub>It might need an extra digest in tests, but this would be the least of the BCs :stuck_out_tongue: </sub>